Yosemite/Yosemite/Actions/AppSettingsAction.swift

```diff
@@ -136,6 +136,14 @@ public enum AppSettingsAction: Action {
     ///
     case setStripeInPersonPaymentsSwitchState(isEnabled: Bool, onCompletion: (Result<Void, Error>) -> Void)
 
+    /// Loads a boolean flag indicating whether the Stripe extension is selected for a given store.
+    ///
+    case loadStripeExtensionAvailability(siteID: Int64, onCompletion: (Result<Bool, Error>) -> Void)
```

Comment:
Ideally, in the mid to long term, this should load and save a custom type, not a boolean flag (see comment above)